sapi/cli/tests/ext_loading.phpt

```diff
@@ -7,6 +7,8 @@ if (!file_exists($extDir . '/opcache.so') && !file_exists($extDir . '/php_opcach
     die('skip Opcache shared object not found in extension_dir');
 }
 ?>
+--SKIPIF--
+<?php if (getenv('SKIP_ASAN')) die('xfail LSAN bug on Clang 14 marks this as leaking'); ?>
```

Comment:
Oh, and also, it doesn't report a leak, it crashes.

Reply:
Oh, and also use xleak please :smile: Sorry for the fragmented comments.

Reply:
Right, also we have trailing whitespaces in the test output so... changed that too